Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -358,27 +358,49 @@ PhysicalParticleContainer::AddPlasmaFromFile(ParticleReal q_tot,
             weight = w * w_unit;
         }
 
+        // TODO: Add option for different levels of injection
```

Comment:
Can you please help me what this refers to? I am just confused because we inject/run/store particles always on the finest level of the refined mesh. I assume this currently injects particles always to the coarsest grid?

Reply:
I am not sure of the future, it is a possible hypothesis, so I wanted to leave the TODO in case anyone else would need to update it later.

Reply:
Sorry, I missed the part where I replied to your question. Yes, this only injects to the coarsest grid. But later we might want to inject to other grids.

Reply:
Okay, the comment was just not clear to me. Maybe
```suggestion
        // TODO: This currently only injects particles on the coarsest grid
```
is more clear?

Reply:
Hum, you are right. It was not clear! But yours is not really a todo... What do you think of the new option?

Reply:
`Note: `/`Fixme: `? :D

Reply:
Great idea!!